docs/topics/js/js-to-kotlin-interop.md

```diff
@@ -33,8 +34,9 @@ Check the article on [JavaScript Modules](js-modules.md) for more information on
 
 ## Package structure
 
-Kotlin exposes its package structure to JavaScript, so unless you define your declarations in the root package,
-you have to use fully qualified names in JavaScript. For example:
+For most of the module systems, Kotlin exposes its package structure to JavaScript,
```

Comment:
I guess it's worth mentioning which module systems, since Kotlin supports four only (not considering PLAIN).  
Plus, you say ESM is an exception, so CJS, AMD and UMD are the one with the package structure.